docs/source/troubleshooting/tracing.rst

```diff
@@ -0,0 +1,106 @@
+==================================================
+Tracing on NI Linux Real-Time
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Introduction
+============
+
+When debugging latency issues, race conditions, or hangs on real-time embedded targets, it is useful to know what the kernel/OS was doing just prior to the error condition. Acquiring and analyzing a kernel event or function trace can provide insights into latency spikes, race conditions, lock contention, priority inversions, missed loop iterations and so on.
+
+This documents covers using the kernel ftrace system.
+
+ftrace
+======
+
+On NILRT systems the kernel debugfs is mounted at `/sys/kernel/debug`. The ftrace subsystem can be controlled via pseudo-files at `/sys/kernel/debug/tracing`. For a quick description of the configuration knobs available, see `/sys/kernel/debug/tracing/README`.
```

Comment:
In RST, the single-backtick (`) marks text for "domain-specific interpretation". In this case, sphinx just makes it _italics_. Inline literals are marked up with double-backticks (``). (I forget this every time too.)

In this line, and throughout the document, you should use the latter for commands, program names, and paths.